Templates/CreateDatabase.sqltemplate

```diff
@@ -0,0 +1,112 @@
+/* Template Args:
+   1. Database Name
+ */
+USE [master]
+GO
+
+CREATE DATABASE [%[1]s]
+ CONTAINMENT = NONE
+ WITH CATALOG_COLLATION = DATABASE_DEFAULT, LEDGER = OFF
+GO
+
+IF (1 = FULLTEXTSERVICEPROPERTY('IsFullTextInstalled'))
+begin
+EXEC [%[1]s].[dbo].[sp_fulltext_database] @action = 'disable'
+end
+GO
+
+ALTER DATABASE [%[1]s] SET ANSI_NULL_DEFAULT OFF
```

Comment:
Scrap what I said before about a whitelist. I didn't realise this was generated from a template.
In this case, most of this can go -- it's mostly useless, but certainly a lot of it is only supported on newer databases.